docs/reference-cli.md

```
@@ -319,6 +343,146 @@ tokmd check-ignore src/main.rs target/release/myapp
+# Compute then gate with specific preset
+tokmd gate --preset health
+
+# Gate with JSON output for CI parsing
+tokmd gate --format json
+
+# Fail fast on first error
+tokmd gate --fail-fast
+```
+
+**Using inline rules in tokmd.toml**:
+```toml
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)